### ISSUE TITLE
Postal - Nodejs client - send error: Error: getaddrinfo ENOTFOUND https://postal.yourdomain.com

### ISSUE THREAD

**Issue by ramioooz** [postalhq/postal#1354]
Hello developers,

I set my postal server, and everything is working just fine
now I am trying to send emails from my node js app 
I am using node js client and following instructions found here:
https://github.com/krystal/postal-node

now when I send an email I am getting an error like this:
`Error: getaddrinfo ENOTFOUND https://postal.yourdomain.com`
of course, yourdomain.com should be replaced with my read domain name.

your help is much appreciated.
Thank you

**Comment by ramioooz** [comment]
I solved it.
in clients/nodejs documentation found here: 
https://github.com/krystal/postal-node

it says I should add host name like this: 
`var client = new Postal.Client('https://postal.yourdomain.com', 'your-api-key');`
I tried that, but it produced the error.
so I write the host name with out https:// . so the line now looks like: 
`var client = new Postal.Client('postal.yourdomain.com', 'your-api-key');`

now it is working as it should.
I hope this helps you too.

Thanks
